### ISSUE TITLE
Turn angle indicator not working WT g3000 0.7.5

### ISSUE THREAD

**Issue by roberwinkler** [Working-Title-MSFS-Mods/fspackages#1401]
Since Sim Update 6 the turn angle indicator in the PFD of the TBM 930 is not working anymore. Without mod it works flawless....

**Comment by 39crauds33** [comment]
I have same issue.  I restarted flightsim without G3000 mod but with TBM mod and the AI functioned normally so it is not the TBM mod nor the update but the G3000 mod with the TBM mod causing the problem.  G3000 (G5000) mod works in CJ4.  Will try G3000 in Longitude (if it still works after SU6).

**Comment by 39crauds33** [comment]
Confirmed the same issue with the Longitude.

**Comment by UberGeekDude** [comment]
A bit more clarity. In Longitude: the PFD Attitude indicator showing pitch and roll stuck at zero. As well, the Flight Directory (Pink) is not visible.  Autopilot flies fine.
I am also seeing CAS indicators like Cabin Diff Press, Oil Press, Main Gen, Ignition, AP ON YD OFF and Auto Sel are lit, though I am flying and engine readout in MFD looks normal.
Terrain is visible, speed and altitude, etc. proper indication.

I was flying yesterday, before adding Working Title, and all worked well in the Longitude.  I did not fly the TBM yet today.  
This is the only mod I added.
Hope it helps.
Regards
